app/src/main/res/values/strings.xml

```diff
@@ -1288,7 +1283,11 @@ If there are no signs along the whole street which apply for the highlighted sec
     <string name="quest_parkingType_underground">Underground garage</string>
     <string name="quest_parkingType_multiStorage">Parking garage</string>
 
-    <string name="quest_pedestrian_crossing_island">"Does this pedestrian crossing have an island?"</string>
+    <string name="quest_pedestrian_crossing_signals">"Does this crossing have traffic signals?"</string>
```

Comment:
This would be a good place to clarify that either vehicular or pedestrian signals would count as a yes. Otherwise, without any additional qualification, users will respond inconsistently: some users will think “traffic signals” only means vehicular signals and say no even if there are pedestrian signals, while others will think a crossing controlled only by vehicular signals “has” no signals at all, because no signals are dedicated only to the crossing. Meanwhile, some users will mistakenly think warning lights count as traffic signals, even though we tag them as [`flashing_lights=yes`](https://wiki.openstreetmap.org/wiki/Key:flashing_lights).

Reply:
Heh. Well, but is there a possible wording that is not massively complicated (and thus confusing enough that it also can lead to misunderstandings and hence wrong taggings)?

Reply:
For someone in Spain or Australia, the quest’s icon already makes the intention very clear, because the signal resembles a pedestrian signal and vehicular signals are presumably irrelevant. I was kind of hoping the icon could very by country so that more countries like Argentina, Canada, or the U.S. could benefit from the same cue. But since this isn’t feasible, would it be possible to display a small icon inside or alongside the question, perhaps varying by region?

> “Are there traffic signals like [<img src="https://upload.wikimedia.org/wikipedia/commons/7/74/Japanese_pedestrian_traffic_signal.svg" height="48" alt="two-phase signal" align="center">](https://commons.wikimedia.org/wiki/File:Japanese_pedestrian_traffic_signal.svg) or [<img src="https://upload.wikimedia.org/wikipedia/commons/b/b2/Traffic_lights_dark_green.svg" height="48" alt="vehicular signal" align="center">](https://commons.wikimedia.org/wiki/File:Traffic_lights_dark_green.svg) at this crossing?

> “Are there traffic signals like [<img src="https://upload.wikimedia.org/wikipedia/commons/1/13/MUTCD_Ped_Signal_-_Walk.svg" height="32" alt="walk signal" align="center">](https://commons.wikimedia.org/wiki/File:MUTCD_Ped_Signal_-_Walk.svg) or [<img src="https://upload.wikimedia.org/wikipedia/commons/b/b2/Traffic_lights_dark_green.svg" height="48" alt="vehicular signal" align="center">](https://commons.wikimedia.org/wiki/File:Traffic_lights_dark_green.svg) at this crossing?”

If that’s too complicated, I would pose a much simpler question:

> “Do you have to wait for a traffic signal to cross here?”

I realize you [aren’t keen on requiring any knowledge of traffic laws](https://wiki.openstreetmap.org/wiki/Proposal_talk:Crossing_signalization#Verifiability_problem_with_shared/separate) to solve quests. However, if the user, a pedestrian, doesn’t know what a crossing or intersection signal looks like, then this quest is probably putting them in immediate danger. 😬 More likely, the user knows what the various signals look like, but they don’t know that a flashing warning light doesn’t count as a “yes” for the purpose of this quest, because it’s worded so vaguely.

Reply:
> “Do you have to wait for a traffic signal to cross here?”

Oh, that's good, I will use that, then.